UnitTests/BraintreeVisaCheckoutTests/BTVisaCheckoutCardNonce_Tests.swift

```diff
@@ -0,0 +1,62 @@
+import XCTest
+@testable import BraintreeCore
+@testable import BraintreeVisaCheckout
+
+final class BTVisaCheckoutCardNonce_Tests: XCTestCase {
```

Comment:
It looks like Android has an additional test that we may be missing: https://github.com/braintree/braintree_android/blob/9a20611391e7cc4f7ce33717dfacad4e47b45143/VisaCheckout/src/test/java/com/braintreepayments/api/visacheckout/VisaCheckoutNonceUnitTest.java#L80C17-L80C78